src/query/execute_schema.c

```diff
@@ -8853,6 +8854,24 @@ execute_create_select_query (PARSER_CONTEXT * parser, const char *const class_na
   return error;
 }
 
+/*
+ * has_replication_key_constraint() -
+ *   return : true if the class has a UNIQUE constraint
+ *   class_obj(in): class object pointer
+ */
+bool
+has_replication_key_constraint (DB_OBJECT * class_obj)
+{
+  DB_CONSTRAINT *c = db_get_constraints (class_obj);
+
+  if (c == NULL)
+    {
+      return false;
+    }
+
+  return classobj_has_class_rk_constraint (c);
```

Comment:
classobj_has_class_rk_constraint 함수에 전달될 c가 NULL이라면 해당 함수에서 false를 반환하는 것으로 보입니다.
간결하게 아래처럼 해도 되지않을까요..?

```suggestion
  return classobj_has_class_rk_constraint (db_get_constraints (class_obj));
```

Reply:
@H2SU 나이스캐치입니다. 말씀하신데로 리팩토링해보니 has_replication_key_constraint()함수도 필요없을것같아 해당함수를 삭제했고, execute에서 바로 classobj_has_class_rk_constraint()를 호출하도록했습니다.